PURL-TYPES.rst

```diff
@@ -582,6 +582,48 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+YOCTO Linux
+-----------
+``yocto`` for Yocto Linux recipes:
+
+There is no default package repository. The classifier `repository_url` is optional.
+
+:namespace: The namespace is the name of the layer which provides the recipe. The layer name as
+specified in the `BBFILE_COLLECTIONS` variable in `conf/layer.conf` of the layer.
+:name: The name of the package also known as [PN](https://docs.yoctoproject.org/ref-manual/variables.html#term-PN) in a yocto recipe.
+:version: The version of the package also known as [PV](https://docs.yoctoproject.org/ref-manual/variables.html#term-PV) in a yocto recipe.
+
+Qualifiers
+^^^^^^^^^^
+All qualifiers should be percent-encoded.
+
+``repository_url``
+  The GIT URL of the layer.
+  In example:
+
+  * ``https%3A%2F%2Fgit.openembedded.org%2Fopenembedded-core``
+  * ``https%3A%2F%2Fgithub.com%2Fakuster%2Fmeta-odroid``
+
+  The protocol can be ``https``, ``ssh``  or ``git`` and is mandatory.
```

Comment:
Does this have to be mandatory? It seems like it _could_ be deduced from the namespace

Reply:
Ah, I misunderstood this, see below

Reply:
But this means that it can be misunderstood by someone else, too. What would be a better way to write it?

> The URL scheme is mandatory. The URL schema can be: `https`, `http`, `ssh`, `git` ...

Reply:
Maybe the above to clarify `repository_url` is optional, and then:

"The URL scheme is mandatory and must be one of `https`, `http`, `ssh`, or `git`."

?